test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -760,7 +745,8 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     It "Validate Invoke-WebRequest default ContentType for CustomMethod POST" {
 
-        $command = "Invoke-WebRequest -Uri 'http://httpbin.org/post' -CustomMethod POST -Body 'testparam=testvalue'"
+        $uri = Get-WebListenerUrl -Test 'Post'
```

Comment:
The test header say "default ContentType" but why the test don't check a content type name?

Reply:
Default content type as in `-ContentType` is not specified. see the line below this.

Reply:
I think I misinterpreted the question. The headline says we're checking the default content type, but inside the test we're not doing it. I expect something like  
```
 $jsonContent.headers.'Content-Type' | Should Be "default-content-type-value" 
```

Reply:
I will take another look at the surrounding code and try to determine what this test was really looking for.

Reply:
I'm not 100% sure what this is testing for, so i added the Headers test. 
Fixed